package.json

```diff
@@ -41,14 +31,14 @@
     "remark-html": "^15.0.1",
     "remark-parse": "^10.0.1",
```

Comment:
Since we move to vite plugin I think we can remove pug from package.json dependency. I can't find usage of this lib in project

Reply:
We use `remark-parse` in `scripts/build-api.js`

Reply:
@ai, I mean `pug` in dependency list. I couldn't leave a comment on the desired line 😦, so I wrote to the nearest available place